lambda_src/batch_locking_backends/dynamodb.py

```diff
@@ -0,0 +1,94 @@
+import os
+from typing import Dict, Text, List, Any
+import boto3
+from json import dumps
+from hashlib import md5
+
+from botocore.exceptions import ClientError
+from ..utils import LOG, ResponseType
+
+AWS_REGION = os.environ.get(
+    "AWS_REGION", "us-west-2"
+)  # Placeholder while in dev TODO: change as variable/header
+DYNAMODB_TABLE = os.environ.get("DYNAMODB_TABLE_NAME")
+DYNAMODB_RESOURCE = boto3.resource("dynamodb", region_name=AWS_REGION)
+if DYNAMODB_TABLE:
+    table = DYNAMODB_RESOURCE.Table(DYNAMODB_TABLE)
+
+TTL = 1800
```

Comment:
let's default to 24h and allow for an environment variable to override this

Reply:
Sure 👍